src/services/array-functions.js

```diff
@@ -1,48 +1,70 @@
 
-//in the function map, create a new array and store in a variable
-//loop theArray and call the fnc for each thing in the array, 
-//     passing in the item from the current loop into the call to fnc
+//in the function map, create a new array '[]'and store in a variable
+//loop 'for' theArray and call 'fnc()' the fnc for each thing in the array,
+//     passing 'push' in the item from the current loop into the call to fnc
 //add the returned value from fnc to the new array
 //return the new array
-export function map(theArray, fnc){
+export function map(names, fnc){
+  var mapped = [];
+  for (var i = 0; i < names.length; i++){
+    mapped.push("Hello " + names[i]);
```

Comment:
double check this code.
you edited the comment to demonstrate what to code, but you didn't actually code it fnc()
its similar to filter but there is no if statement